test_user.py

```diff
@@ -0,0 +1,76 @@
+"""test methods for the user class"""
+import unittest
+from user import User
+
+class TestUser(unittest.TestCase):
```

Comment:
Your tests have not included testing if the links work eg self.client.post(<url>)